### PR TITLE
Rhel7 branch bridged bond

### PR DESCRIPTION
**Pull request by rvykydal**
Add support for bridging bond devices (#1321288)

### CODE REVIEW

**Comment by jkonecny12** [comment]
Jenkins error:
```
************* Module ../dracut/parse-kickstart
W0621:619,16: update_virtual_slave_ifcfg: Redefining name 'line' from outer scope (line 654)
```

File: dracut/parse-kickstart [hunk 1]
```diff
@@ -553,6 +553,10 @@ def ksnet_to_ifcfg(net, filename=None):
             ifcfg['BRIDGING_OPTS'] = " ".join(keyvalues)
 
         for i, slave in enumerate(net.bridgeslaves.split(","), 1):
+            # Handle virtual slaves created by preceding network commands (eg bond)
+            if update_virtual_slave_ifcfg(slave,
+                    setting="BRIDGE=%s" % dev):
```
Comment (jkonecny12) [inline dracut/parse-kickstart]: Could you please remove the newline here, I think it will be more readable without it .

**Comment by jkonecny12** [comment]
Are you planning to send this PR to master too?

File: dracut/parse-kickstart (hunk 1 above)
Comment (rvykydal) [inline dracut/parse-kickstart]: Yes

**Comment by rvykydal** [comment]
I haven't tried with master yet. I guess I'll make another PR for master.

**Comment by rvykydal** [comment]
Force pushed update that should fix the pylint error and remove the newline as requested.

**Comment by jkonecny12** [comment]
Looks good to me now.

**Pull request merged**